docs/content/ecs/logging.md

```diff
@@ -8,6 +8,7 @@ The SpatialOS GDK for Unity uses a custom `ILogDispatcher` interface instead of
 
 *  `LoggingDispatcher`, which simply logs to the Unity console
 *  `ForwardingDispatcher`, which logs to the Unity console and sends it to the SpatialOS Console
+    * Note: By default, `LogType.Log` is not sent to SpatialOS Console. This can be changed by instantiating the `ForwardingDispatcher` with different `minimumLogLevel` parameter
```

Comment:
> with *a* different

missing period at end of line

Reply:
add an example?

Reply:
I don't think that there should be an example here. Right below is a link to a section which talks about creating a dispatcher. Maybe I should rename it to "Modifying" instead of "Creating".